.travis.yml

```diff
@@ -69,7 +80,16 @@ script:
 
 after_success:
  - ccache -s
- - if [[ ${COVERAGE} ]]; then pushd build && bash <(curl -s https://codecov.io/bash) && popd; fi
+ - test "${TRAVIS_BUILD_STAGE_NAME}" = "Test" || travis_terminate 0
```

Comment:
Isn't this always going to terminate early?

Reply:
No, the logic is: is it the test stage or terminate.